libyul/backends/wasm/WasmCodeTransform.cpp

```diff
@@ -333,6 +336,7 @@ wasm::FunctionDefinition WasmCodeTransform::translateFunction(yul::FunctionDefin
 		fun.parameters.push_back({param.name.str(), translatedType(param.type)});
 	for (auto const& retParam: _fun.returnVariables)
 		fun.locals.emplace_back(wasm::VariableDeclaration{retParam.name.str(), translatedType(retParam.type)});
+	yulAssert(_fun.returnVariables.size() <= 1, "Only up to 1 return variable supported.");
```

Comment:
I think this is incorrect. The code below handles multiple return variables just fine - by returning stuff using globals.

Reply:
Are those CLI tests failing proving this? Can we have proper unit tests for this? :)

Reply:
> Are those CLI tests failing proving this?

They only say that process did not return 0. That's what would happen if this assert failed. But any other assertion failing would have the same effect.

Maybe we could get the command line tests to test exit code last so that we get more informative output if they fail?

> Can we have proper unit tests for this? :)

Maybe #9143 needs a little push :)

Reply:
> The code below handles multiple return variables just fine - by returning stuff using globals.

Sure, this code was added before you implemented that. Will remove.